api/httpjson/RPCserver.go

```diff
@@ -312,3 +312,26 @@ func (s *RPCServer) SetLocalNode(ln *node.LocalNode) {
 func (s *RPCServer) GetNetNode() *node.LocalNode {
 	return s.GetLocalNode()
 }
+
+func (s *RPCServer) setRPCTlsInfo() error {
```

Comment:
It's better to set local node when creating local node because local node info might be cached by neighbors once created and client might get outdated info.

Reply:
we can not set it correctly before we obtain the certs, and we obtain certs functions were execution asynchronous